src/events/deposits.upsert.service.ts

```diff
@@ -131,4 +132,23 @@ export class DepositsUpsertService {
 
     return deposits;
   }
+
+  async mismatchedDepositCount(): Promise<number> {
+    const result = await this.prisma.$queryRawUnsafe<{ count: number }[]>(
+      `
+      SELECT
+        COUNT(*) AS count
```

Comment:
nit: I think you can omit `AS count`, as Postgres defaults `COUNT` to this column